Studio-Lovelies/GG-JointJustice-Unity
# Adds ability to present evidence during investigation sections

## Summary
Adds ability to present evidence during investigation sections. Also adds a contribution guide.

## Testing instructions
1. Open the `Game` scene
2. Set the `GameStarter` to `InvestigationUI`
3. Open the `Talk` submenu and count the available options
4. Go back to the investigation main menu
5. Open the `Present` submenu and present the `Plumber Invoice`
6. Notice one of two generic failure scripts playing
7. Open the `Talk` submenu and notice the same amount of options being available
8. Go back to the investigation main menu
9. Open the `Present` submenu and present `Jory Sr's Letter`
10. Open the talk submenu and notice one more available option
11. Go back to the investigation main menu
12. Open the `Present` submenu and present the `Switch`
13. Open the talk submenu and notice one fewer available option

## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  #456
